srcpkgs/NetworkManager-l2tp/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'NetworkManager-l2tp'
+pkgname=NetworkManager-l2tp
+version=1.8.2
+revision=1
+build_style=gnu-configure
+configure_args="--runstatedir=/run"
+hostmakedepends="pkg-config intltool glib-devel"
+makedepends="gtk+3-devel libglib-devel libsecret-devel network-manager-applet-devel
+libressl-devel nss-devel ppp-devel"
+depends="strongswan xl2tpd ppp"
+short_desc="NetworkManager VPN plugin for L2TP/IPsec"
+maintainer="Douglas Kosovic <doug@uq.edu.au>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/nm-l2tp/NetworkManager-l2tp"
+distfiles="https://github.com/nm-l2tp/NetworkManager-l2tp/releases/download/${version}/${pkgname}-${version}.tar.xz"
+checksum=1d80986dc88951e298446833d314bab7af0a933e736960d6b3b025e6b7faca94
+
+do_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		PPP_VERSION=$(sed -n 's/^\#define\s*VERSION\s*"\([^\s]*\)"$/\1/p' /usr/${XBPS_CROSS_TRIPLET}/usr/include/pppd/patchlevel.h)
+	else
+		PPP_VERSION=$(sed -n 's/^\#define\s*VERSION\s*"\([^\s]*\)"$/\1/p' /usr/include/pppd/patchlevel.h)
```

Comment:
An explanation of what this does in a comment above would be *nice* but not entirely necessary.

Reply:
I'll try and come up with something.

Unfortunately the upstream ppp package doesn't use pkgconfig yet.  For the PPP based NetworkManager VPNs, Debian uses `dh_ppp --plugin-dir`, OpenSUSE has `%requires_eq ppp`, Fedora does a similar `sed` on `/usr/include/pppd/patchlevel.h`.

For the `depends`, I would have liked to have had something like `ppp=${PPP_VERSION}`, to enforce that the package should get rebuilt whenever a new version of the ppp package comes along which uses a pppd plugin dir location based on the ppp version number.

Reply:
That would be optimal. I came across a similar situation for another package a few months back. I just ended up adding a comment to the beginning of the template that the package should be revbumped when such a new version is released.